src/services/UserService.ts

```diff
@@ -1,12 +1,12 @@
-import { Injectable } from '@nestjs/common';
+import { Injectable, HttpException, HttpStatus } from '@nestjs/common';
 import * as uuid from 'uuid';
 import User from '@/data-transfer-object/UserDto';
 import users from '@/mocks/users';
 import EmailService from '@/services/EmailService';
-
+let usersData = users;
```

Comment:
Can an extra variable be avoided?

Reply:
it's for update and delete that import data, we could not write or edit the imported `users` data